### PR TITLE
RDCC-2035: Using PRD S2S Token when calling UP #931

### PR DESCRIPTION
**Pull request by AdilOoze**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-2035

### Change description ###

Using PRD S2S Token when calling UP

Linked to: https://github.com/hmcts/rd-user-profile-api/pull/581

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/rd-professional-api/values.preview.template.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ java:
     POSTGRES_USERNAME: "{{ .Values.postgresql.postgresqlUsername}}"
     POSTGRES_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
     LAUNCH_DARKLY_ENV: "preview"
+    USER_PROFILE_URL: http://rd-user-profile-api-pr-581.service.core-compute-preview.internal
```
Comment (shreedhar-hmcts) [inline charts/rd-professional-api/values.preview.template.yaml:10]: need to be removed?

File: charts/rd-professional-api/values.preview.template.yaml [hunk 2]
```diff
@@ -17,5 +18,4 @@ java:
       secrets:
         - LD-SDK-KEY
         - AppInsightsInstrumentationKey
-
-
+        - s2s-secret
```
Comment (vijayrajagopalan-hmcts) [inline charts/rd-professional-api/values.preview.template.yaml:21]: Please use professional-api-s2s-secret. s2s-secret will be deleted in the future.

File: charts/rd-professional-api/values.preview.template.yaml (hunk 1 above)
Comment (AdilOoze) [reply charts/rd-professional-api/values.preview.template.yaml:10]: Yes, can be removed now that testing is completed

Comment (AdilOoze) [reply charts/rd-professional-api/values.preview.template.yaml]: Change completed here: https://github.com/hmcts/rd-professional-api/pull/954/commits/b142797a575f44d1b8227e9976a872b68c97e4aa

File: charts/rd-professional-api/values.preview.template.yaml (hunk 2 above)
Comment (AdilOoze) [reply charts/rd-professional-api/values.preview.template.yaml]: Change completed here: https://github.com/hmcts/rd-professional-api/pull/954/commits/db195119a1200e625b50e0a477708b50f07c17ac

**Review by SujeethKumarAri — APPROVED** [review]

**Review by shreedhar-hmcts — APPROVED** [review]

**Review by vijayrajagopalan-hmcts — APPROVED** [review]

**Review by nitishsingla1234 — APPROVED** [review]

**Pull request merged**